src/inspector/models/abstractinspectormodel.cpp

```diff
@@ -416,6 +416,9 @@ mu::engraving::Sid AbstractInspectorModel::styleIdByPropertyId(const mu::engravi
     mu::engraving::Sid result = mu::engraving::Sid::NOSTYLE;
 
     for (const mu::engraving::EngravingItem* element : m_elementList) {
+        IF_ASSERT_FAILED(element) {
```

Comment:
I can't reproduce the crash these lines are avoiding - I think it's safe to remove them.

Reply:
I added this lines before I solved the problem within importmusicxmlpass2.cpp. Therefore I can delete them.

About:

> I can reproduce the crash, but I think it might be better to fix the issue further upstream.

I haven't been able to reproduce now the crash. Could yo provide me with an example to test it and solve it please ?

Thanks

Reply:
Sorry to bother @miiizen , could you please tell me how you can reproduce the crash ?